test/DynamoCoreTests/DynamoModelTestBase.cs

```diff
@@ -37,6 +39,8 @@ protected override DynamoModel GetModel()
         public override void Setup()
         {
             base.Setup();
+
+            originalBuiltinPackagesDirectory = PathManager.BuiltinPackagesDirectory;
```

Comment:
@pinzart90 I guess this is better... definitely more magical :) Maybe update the comment on `BuiltinPackagesDirectory` to note it's reset only on `DynamoModelTestBase` - one unfortunate issue we have lots of test base classes.